DTS-STN/canadian-dental-care-plan
# AB#3105 -- Adding service for retrieving client friendly statuses

### Description
Status checker will utilize this service to display the (English/French) text for a status of a CDCP application.

### Related Azure Boards Work Items
[AB#3105](https://dev.azure.com/DTS-STN/Canada%20Dental%20Care%20Plan/_workitems/edit/3105)

### Checklist

### Test Instructions
Add this block of code somewhere and verify that the statuses are logged:
```typescript
  const clientFriendlyStatuses = await getLookupService().getAllClientFriendlyStatuses();
  console.log(JSON.stringify(clientFriendlyStatuses));
```
